layout-server/src/test/java/org/finos/vuu/layoutserver/integration/LayoutIntegrationTest.java

```diff
@@ -109,40 +110,6 @@ void getMetadata_singleMetadataExists_returnsMetadata() throws Exception {
                         is(layout.getMetadata().getBaseMetadata().getUser())));
     }
 
-    @Test
-    void getMetadata_multipleMetadataExists_returnsAllMetadata() throws Exception {
-        Layout layout1 = createDefaultLayoutInDatabase();
-        Layout layout2 = createDefaultLayoutInDatabase();
-        layout2.setDefinition("Different definition");
-        layout2.getMetadata().getBaseMetadata().setName("Different name");
-        layout2.getMetadata().getBaseMetadata().setGroup("Different group");
-        layout2.getMetadata().getBaseMetadata().setScreenshot("Different screenshot");
-        layout2.getMetadata().getBaseMetadata().setUser("Different user");
-        layoutRepository.save(layout2);
-
-        assertThat(layoutRepository.findById(layout1.getId()).orElseThrow()).isEqualTo(layout1);
-        assertThat(layoutRepository.findById(layout2.getId()).orElseThrow()).isEqualTo(layout2);
-
-        mockMvc.perform(get("/layouts/metadata"))
-                .andExpect(status().isOk())
-                .andExpect(jsonPath("$[0].name",
-                        is(layout1.getMetadata().getBaseMetadata().getName())))
-                .andExpect(jsonPath("$[0].group",
-                        is(layout1.getMetadata().getBaseMetadata().getGroup())))
-                .andExpect(jsonPath("$[0].screenshot",
-                        is(layout1.getMetadata().getBaseMetadata().getScreenshot())))
-                .andExpect(jsonPath("$[0].user",
-                        is(layout1.getMetadata().getBaseMetadata().getUser())))
-                .andExpect(jsonPath("$[1].name",
-                        is(layout2.getMetadata().getBaseMetadata().getName())))
-                .andExpect(jsonPath("$[1].group",
-                        is(layout2.getMetadata().getBaseMetadata().getGroup())))
-                .andExpect(jsonPath("$[1].screenshot",
-                        is(layout2.getMetadata().getBaseMetadata().getScreenshot())))
-                .andExpect(jsonPath("$[1].user",
-                        is(layout2.getMetadata().getBaseMetadata().getUser())));
-    }
-
```

Comment:
How come we're getting rid of this test?

Reply:
ooops! My bad! Actually we still need this!